tests/e2e/specs/shopper/mini-cart.test.js

```diff
@@ -7,12 +12,33 @@ const block = {
 	name: 'Mini Cart Block',
 };
 
+const options = getDefaultOptions();
+
+const clickMiniCartButton = async () => {
+	await page.hover( '.wc-block-mini-cart__button' );
+
+	await expect( page ).not.toMatchElement(
+		'.wc-block-mini-cart__drawer.is-loading'
+	);
+
+	await page.click( '.wc-block-mini-cart__button' );
+};
+
 if ( process.env.WOOCOMMERCE_BLOCKS_PHASE < 3 ) {
 	// eslint-disable-next-line jest/no-focused-tests
 	test.only( `skipping ${ block.name } tests`, () => {} );
 }
 
 describe( 'Shopper → Mini Cart', () => {
+	beforeAll( async () => {
+		setDefaultOptions( { ...options, timeout: 5000 } );
+	} );
```

Comment:
let's add a comment to highlight the details about how this is required because of the script being lazy loaded on hover action over the Mini Cart icon